<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionsTable</class>
 <widget class="QWidget" name="ActionsTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ActionsTable</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QTableWidget" name="main_table">
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PID</string>
      </property>
      <property name="whatsThis">
       <string extracomment="PID is short for Process ID"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Process name</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="path_filter_edit">
     <property name="placeholderText">
      <string>Filter by path</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
